smoketest/tests/lock_tokens.rs

```diff
@@ -67,7 +67,16 @@ async fn test_lock_tokens() {
         .unwrap();
     assert_eq!(receipt.status.unwrap().as_u64(), 1u64);
 
-    let recipient = MultiLocation::default().encode();
+    // Ferdie 
+    // DE14BzQ1bDXWPKeLoAqdLAm1GpyAWaWF1knF74cEZeomTBM
+    // 0x1cbd2d43530a44705ad088af313e18f80b53ef16b36177cd4b77b846f2a5f07c
+    let recipient = MultiLocation {
+        parents: 0,
+        interior: X1(Junction::AccountId32{
+            network: None,
+            id: [0x1c, 0xbd, 0x2d, 0x43, 0x53, 0x0a, 0x44, 0x70, 0x5a, 0xd0, 0x88, 0xaf, 0x31, 0x3e, 0x18, 0xf8, 0x0b, 0x53, 0xef, 0x16, 0xb3, 0x61, 0x77, 0xcd, 0x4b, 0x77, 0xb8, 0x46, 0xf2, 0xa5, 0xf0, 0x7c],
```

Comment:
please rather use `hex_literal:hex!` macro here.

Reply:
resolved in [90691c8](https://github.com/Snowfork/snowbridge/pull/846/commits/90691c836b69be19a7c2f0da307d1c4af927344b)